plugins/module_utils/network/vyos/argspec/lag_interfaces/lag_interfaces.py

```diff
@@ -41,12 +45,19 @@ def __init__(self, **kwargs):
                 "arp_monitor": {
                     "options": {
                         "interval": {"type": "int"},
-                        "target": {"type": "list", "elements": "str"},
+                        "target": {
+                            "elements": "str",
+                            "type": "list",
+                        },
                     },
                     "type": "dict",
                 },
                 "hash_policy": {
-                    "choices": ["layer2", "layer2+3", "layer3+4"],
+                    "choices": [
+                        "layer2",
+                        "layer2+3",
+                        "layer3+4",
+                    ],
```

Comment:
There are more options:
```
vyos@client-11# set interfaces bonding bond0 hash-policy 
Possible completions:
   layer2       use MAC addresses to generate the hash (default)
   layer2+3     combine MAC address and IP address to make hash
   layer3+4     combine IP address and port to make hash
   encap2+3     combine encapsulated MAC address and IP address to make hash
   encap3+4     combine encapsulated IP address and port to make hash
                

```

Reply:
Argspec are auto-generated from the `DOCUMENTATION` in the modules themselves, please focus on those unless they're inconsistent.

For LAG that module is still only 1.2, so those options will show up when they're supported. The goal of this PR is to get the documentation differences from the current code resolved.